### ISSUE TITLE
[BUG] Docker image 5.3.0 breaks custom index.html

### ISSUE THREAD

**Issue by RedPine404** [linuxserver/docker-librespeed#26]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When deploying librespeed with container tag 5.3.0 (latest), my custom template in /config/www/index.html is not served. On a clean build, template examples are also not generated in /config/www

### Expected Behavior

My template in /config/www/index.html should be served and templates should be generated in /etc/www on a clean build.

This is working when I roll back to version 5.2.5.

### Steps To Reproduce

Deploy container lscr.io/linuxserver/librespeed:5.3.0
check logs
check /config/www/index.html

### Environment

```markdown
- OS: Ubuntu 22.04.3 LTS
- How docker service was installed: apt package, docker version 1.5-2
```

### CPU architecture

x86-64

### Docker creation

```bash
librespeed:
    image: lscr.io/linuxserver/librespeed:5.3.0 #latest
    container_name: librespeed
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Etc/UTC
      - PASSWORD=PASSWORD
    volumes:
      - $PWD/librespeed:/config
    ports:
      - "80:80"
    restart: unless-stopped
```

### Container logs

```bash
[migrations] started
[migrations] 01-nginx-site-confs-default: skipped
[migrations] 02-default-location: skipped
[migrations] done
───────────────────────────────────────

      ██╗     ███████╗██╗ ██████╗ 
      ██║     ██╔════╝██║██╔═══██╗
      ██║     ███████╗██║██║   ██║
      ██║     ╚════██║██║██║   ██║
      ███████╗███████║██║╚██████╔╝
      ╚══════╝╚══════╝╚═╝ ╚═════╝ 

   Brought to you by linuxserver.io
───────────────────────────────────────

To support the app dev(s) visit:
LibreSpeed (Liberapay): https://liberapay.com/fdossena/donate
LibreSpeed (Paypal): https://www.paypal.me/sineisochronic

To support LSIO projects visit:
https://www.linuxserver.io/donate/

───────────────────────────────────────
GID/UID
───────────────────────────────────────

User UID:    1000
User GID:    1000
───────────────────────────────────────

using keys found in /config/keys
ln: failed to create symbolic link '/app/www/public/index.html': File exists
cp: cannot stat '/app/www/public/example*.html': No such file or directory
[custom-init] No custom files found, skipping...
[ls.io-init] done.
```

**Comment by deltabweb** [comment]
I'm having the same issue

**Comment by hogdanish** [comment]
This is still an issue as of 2/13/2024.

```
failed to create symbolic link '/app/www/public/index.html': File exists
cp: cannot stat '/app/www/public/example*.html': No such file or directory
```

**Comment by pcislocked** [comment]
Can confirm it's still an issue. Version 5.2.5-ls168 seems fine.

**Comment by joe-eklund** [comment]
Can confirm `latest` is still broken. I am stuck on linuxserver/librespeed:5.2.5-ls168 until this is fixed.

**Comment by zinkscott** [comment]
How is this still an issue? Manually overwriting the HTML as RO or using an older container image is not ideal, much like the reliability of this image.

**Comment by aptalca** [comment]
Upstream keeps the static app files mixed in with user modifiable files, which makes our init a hacky one. On top of that, they keep changing the file/folder structures and keep breaking our init for updates.

I used to use this in production when we first published the docker image so I kept up with the changes then but I no longer use it so I haven't stayed on top of the cat and mouse game.

**Comment by liOnux-fr** [comment]
Hi. Same issue on ARM64.
But while waiting for the bug to be fixed, map these volumes : "/path/to/librespeed/www/public:/app/www/public" and customize the index.html found there.

**Comment by Zebradil** [comment]
@aptalca I believe this issue can be fixed by #31. Could you please take a look at this PR? There are just two changed lines.
